app/assets/stylesheets/application.scss

```diff
@@ -84,114 +86,6 @@ main {
         @include govuk-focused-text;
       }
     }
-
-    .in-manual-search {
-      position: relative;
-      margin: govuk-spacing(6) 0 govuk-spacing(3);
-
-      label {
-        padding-bottom: govuk-spacing(1);
-        display: inline-block;
-      }
-
-      .in-manual-search-box {
-        width: 86%;
-        width: calc(100% - 47px);
-        border: none;
-        padding: 9px 7px;
-        height: 16px;
-
-        &.focus,
-        &:focus {
-          position: relative;
-          z-index: 1;
-          outline-color: $govuk-focus-colour;
-          box-shadow: inset 0 0 0 3px govuk-colour("black");
-        }
-
-        @include govuk-media-query($from: tablet) {
-          padding: 7px;
-          height: 20px;
-        }
-      }
-
-      button {
-        width: 34px;
-        height: 34px;
-        border: 1px solid govuk-colour("black");
-        border-width: 0 0 0 1px;
-        border-left-color: $manual-search-button-border-colour;
-        overflow: hidden;
-        text-indent: -5000px;
-        cursor: pointer;
-        color: govuk-colour("white");
-        background: govuk-colour("black") image-url("search-button.png") no-repeat 0 50%;
-        position: absolute;
-
-        border-radius: 0;
-
-        @include govuk-device-pixel-ratio {
-          background-size: 52.5px 35px;
-          background-position: 100% 50%;
-        }
-
-        &:focus {
-          outline-color: $govuk-focus-colour;
-        }
-      }
-    }
-  }
-
-  .breadcrumb-trail {
-    @include govuk-font(16);
-    margin: govuk-spacing(2) 0 0 0;
-    padding: 0 0 govuk-spacing(2);
-    border-bottom: 1px solid $govuk-border-colour;
```

Comment:
Bottom border flag is present - will be visible once repo gets gem update 24.4

![Screenshot 2021-03-04 at 17 28 41](https://user-
images.githubusercontent.com/71266765/110004253-22fe3a80-7d0f-11eb-9391-1de0e55d85b4.png)